hangover.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/sh -e
 
+# Fill in the blanks
+# Phone numbers with leading country code: +46 012 345 6788
+MY_NUMBER=''
+NUMBER_OF_BOSS=''
+
+# Twilio account information
+TWILIO_ACCOUNT_SID=""
```

Comment:
How about:

MY_NUMBER=${MY_NUMBER:-}

That way you can set them in the script or in your env?